srcs/renderer/shaders/window.frag

```diff
@@ -38,6 +42,21 @@ uint GetValueAt(uint addr)
     return data1B;
 }
 
+Lcd GetLcd(uint line)
+{
+    uint index = line * 5u / 4u;
+    uint offset = line * 5u % 4u;
+
+    Lcd lcd;
+    lcd.lcdc = lcdArray[index + ((offset + 0u) / 4u)][(offset + 0u) % 4u];
+    lcd.scx = int(lcdArray[index + ((offset + 1u) / 4u)][(offset + 1u) % 4u]);
+    lcd.scy = int(lcdArray[index + ((offset + 2u) / 4u)][(offset + 2u) % 4u]);
+    lcd.wx = int(lcdArray[index + ((offset + 3u) / 4u)][(offset + 3u) % 4u]);
+    lcd.wy = int(lcdArray[index + ((offset + 4u) / 4u)][(offset + 4u) % 4u]);
```

Comment:
what the fuck

Reply:
This is PERFORMANCE

Reply:
You convinced me. Changes have been merged into master. This is very high quality code, thank you Mr. Burel.